src/EditorFeatures/Core/Wrapping/ChainedExpression/AbstractChainedExpressionWrapper.cs

```diff
@@ -0,0 +1,309 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Immutable;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.LanguageServices;
+using Microsoft.CodeAnalysis.PooledObjects;
+
+namespace Microsoft.CodeAnalysis.Editor.Wrapping.ChainedExpression
+{
+    /// <summary>
+    /// Finds and wraps 'chained' expressions.  For the purpose of this feature, a chained
+    /// expression is built out of 'chunks' where each chunk is of the form
+    ///
+    /// <c>
```

Comment:
📝 `<c>` is the inline code element. If you mean for these to be separate blocks treated similar to paragraphs, use `<code>`.

Reply:
thanks!

Reply:
resolved.